mmv1/products/apigee/EnvKeystore.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+  name: 'EnvKeystore'
+  base_url: '{{env_id}}/keystores'
+  create_url: '{{env_id}}/keystores'
+  delete_url: '{{env_id}}/keystores/{{name}}'
+  self_link: '{{env_id}}/keystores/{{name}}'
+  input: true
+  description: |
+    An `Environment KeyStore` in Apigee.
+  parameters:
+    - !ruby/object:Api::Type::String
+      name: 'envId'
+      description: |
+        The Apigee environment group associated with the Apigee environment,
+        in the format `organizations/{{org_name}}/environments/{{env_name}}`.
+      required: true
+      input: true
+      url_param_only: true
+    - !ruby/object:Api::Type::String
+      name: 'name'
+      description: |
+        The name of the newly created keystore.
+      input: true
```

Comment:
note: The regex used by the API accepts spaces, so we should test a name that includes them.